src/log/access_log.cc

```diff
@@ -58,8 +58,9 @@ static void mcast_encode(unsigned int *, size_t, const unsigned int *);
 #if USE_FORW_VIA_DB
 
 using HeaderValueCountsElement = std::pair<SBuf, uint64_t>;
+using HeaderValueCountsAllocatorElement = std::pair<const SBuf, uint64_t>;
```

Comment:
Same recommendation here:

```suggestion
using HeaderValueCountsElement = std::pair<const SBuf, uint64_t>;
```

... and revert the HeaderValueCounts change below.